crawl/bazel.py

```diff
@@ -95,6 +96,34 @@ def query_all_libraries(repository_root_path, packages, verbose=False):
 def _normalize_dependency_string_to_group_id_artifact_id(dependency):
     return ":".join(dependency.split(":")[:2]) # remove classifier/package, so we can grab version from artifacts list
 
+def parse_override_file(file_path):
```

Comment:
This method could be very well unit tested e.g. testing a file with comments etc.